src/dbnode/storage/index_block_test.go

```diff
@@ -566,26 +566,41 @@ func TestNamespaceIndexBlockQuery(t *testing.T) {
 		StartInclusive: t0,
 		EndExclusive:   now.Add(time.Minute),
 	}
-	b0.EXPECT().Query(gomock.Any(), q, qOpts, gomock.Any()).Return(true, nil)
+	aggOpts := index.AggregateResultsOptions{}
```

Comment:
nit: could you decouple the tests for the Query and Aggregate. One simple way - copy paste the test you have twice, and make it only do the mocks/calls for Query in one, and Aggregate in the other. That way each is independent of the other. Makes maintenance easier in the long run

Reply:
Yeah, for sure. I didn't want to go ahead and c/p larger tests so figured would be fine to package them; will refactor to use a test setup method

Reply:
Tried with a setup method and it looked really bad; did the test twice instead